examples/hosts/app-integration/external-controller/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const path = require("path");
+const merge = require("webpack-merge");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+// const { CleanWebpackPlugin } = require("clean-webpack-plugin");
```

Comment:
nit

Reply:
I need to bring the new and improved config back from the Hello World repo, will make a note to do this later.